### ISSUE TITLE
Is there a wiki for documentation?

### ISSUE THREAD

**Issue by topresearch** [silexlabs/cloud-explorer#49]
Hello!

I want to see the documentation (not only getting started but also what the functions do) for this project but I can't find it anywhere. How and/or where can I read it?

Regards

**Comment by lexoyo** [comment]
Hi
Glad to see that you continue with us on CE. Consider the issues like a forum to ask us questions. We use CE in different projects so we can show you some code for integration (see silexlabs/Silex)
If you feel like it and @zabojad agrees we can open github wiki for the project...
Best regards
Alex Hoyau

**Comment by topresearch** [comment]
@lexoyo Yes it helps to see how do you integrate CE with other projects, to better know what to do next.

About the documentation, I would try your approach first and, if needed, I'll update this issue.

Regards

**Comment by lexoyo** [comment]
Ok nice

Here is how responsize.org uses CE (it is coded in ecmascript 6, the
upcoming version of javascript)
https://github.com/silexlabs/responsize/blob/master/src/rsz/file-service/file-service.js

And here is the integration in backnode.io
https://github.com/silexlabs/BackNode/blob/b52ae41da425ef30e4beac6b03e9d57d547ca74e/src/hx/backnode/App.hx
https://github.com/silexlabs/BackNode/blob/b52ae41da425ef30e4beac6b03e9d57d547ca74e/src/hx/Externs.hx

And in Silex.me (pure js)
https://github.com/silexlabs/Silex/blob/master/src/js/service/cloud-storage.js





On Mon, Mar 9, 2015 at 11:02 AM, TOP Research <notifications@github.com>
wrote:

> @lexoyo <https://github.com/lexoyo> Yes it helps to see how do you
> integrate CE with other projects, to better know what to do next.
>
> About the documentation, I would try your approach first and, if needed,
> I'll update this issue.
>
> Regards
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/silexlabs/cloud-explorer/issues/49#issuecomment-77826875>
> .
>



-- 
If you want to keep in touch with Silex Labs subscribe here
<http://eepurl.com/MCCDP>

**Comment by topresearch** [comment]
Thank you! :)
